src/main/java/CheckCommand.java

```diff
@@ -0,0 +1,21 @@
+public class CheckCommand extends Command{
+    private int checkInt;
```

Comment:
Perhaps you could consider adding JavaDoc comments for class variables as well? Just a small suggestion!